pages/common/sd.md

```diff
@@ -0,0 +1,19 @@
+# sd
+
+> `sd` is an intuitive find & replace CLI.
+
+- Trim some whitespace using regex:
+
+`{{echo 'lorem ipsum 23   '}} | sd '\s+$' ''`
+
+- Replace words using capture groups:
+
+`{{echo 'cargo +nightly watch'}} | sd '(\w+)\s+\+(\w+)\s+(\w+)' 'cmd: $1, channel: $2, subcmd: $3'`
+
+- Find and replace in a file:
+
+`sd -i {{'window.fetch'}} {{'fetch'}} {{http.js}}`
+
+- Find and replace across a project:
+
+`sd -i {{'from "react"'}} {{'from "preact"'}} $(fd -t f)`
```

Comment:
What's `fd`?

Reply:
A Rust alternative to `find`. [1]

[1] https://github.com/sharkdp/fd

Reply:
I see. Generally, a page should document the functionality of the command itself, instead of introducing combinations of other commands.

In this instance, if we were to include the `fd` here, I'd be inclined to use a regular-old `find`, as most users will have that installed by default.

Reply:
That's a good point. I've updated it to use `find`. Thanks for the feedback!